app-misc/fhem/fhem-5.8.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="A GPL'd perl server for house automation"
+HOMEPAGE="https://www.fhem.de/"
+SRC_URI="https://www.fhem.de/${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
```

Comment:
does not it really work? Why did you restrict everything but amd64 and x86, perlish things seems to work everything

Reply:
Just a mistake, as it should work elsewhere too. Fixed.